### DISCUSSION TITLE
Setting default CircleMarker radius

### DISCUSSION THREAD

**Discussion by TheFlyingPolak** [geoman-io/leaflet-geoman#1435] [Questions & Help] [answered]
Hi! I've been trying to find a way to change the default radius of circle markers drawn by geoman. In leaflet it's very easy to do when a new CircleMarker object is created, just pass the option. However, there doesn't really seem to be a way to do it in geoman.

There is a variable `this._defaultRadius=10` in `L.PM.Draw.CircleMarker.js` but there doesn't seem to be a way to change its value.

Thanks in advance for any suggestions!

**Comment by Falke-Design** [comment, marked as answer]
Hi,
you can use following:

```
map.pm.setGlobalOptions({
	templineStyle: {
  	radius: 30
  },
	pathOptions: {
  	radius: 30
  }
})
```
